pkg/model/manifest.go

```diff
@@ -686,11 +686,11 @@ func (m *Manifest) SanitizeSvcNames() error {
 		}
 	}
 
-	for _, gf := range m.GlobalForward {
-		if shouldBeSanitized(gf.ServiceName) {
-			sanitizedSvcName := sanitizeName(gf.ServiceName)
-			gf.ServiceName = sanitizedSvcName
-			sanitizedServicesNames[gf.ServiceName] = sanitizedSvcName
+	for indx := range m.GlobalForward {
```

Comment:
Can we use `idx` as the index abbreviation, that keeps consistency between the rest of the code.